uint8 orderID
uint8 tugID